orangecontrib/text/widgets/owkeywords.py

```diff
@@ -79,10 +86,20 @@ def callback(i: float, status=""):
                 kw = {"progress_callback": cb}
                 kw.update(scoring_methods_kwargs.get(method_name, {}))
 
-                keywords = func(corpus if needs_tokens else documents, **kw)
-                results.all_keywords[method_name] = keywords
+                kws = func(corpus if needs_tokens else documents, **kw)
+                # None means that embedding completely failed on document
+                # currently it only happens with mbert when connection fails
+                keywords = [kws for kws in kws if kws is not None]
```

Comment:
Minor comment: I'd have `for kw in kws`; because it's singular and also because somebody (who doesn't know that this was fixed in Python 3) would fear that `kws` would be leaked out of list comprehension.

I'm merging as it is; if you want, just push a commit with this change into master. :)

Reply:
It was actually my mistake (probably while making some changes). Agree to the use of the singular. I will fix it on master.